### PR TITLE
refactor:add to_binary trait and fix row_description 

### PR DESCRIPTION
**Pull request by ZENOTME**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

1. add to_binary trait 
2. fix row_description:return the correct format code( we used to return text all the time   

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

### CODE REVIEW

File: src/utils/pgwire/src/pg_extended.rs [hunk 1]
```diff
@@ -81,12 +81,22 @@ impl PgStatement {
...
 
+        let row_description: Vec<PgFieldDescriptor> = if result_format {
+            let mut row_description = self.row_description.clone();
+            row_description
+                .iter_mut()
+                .for_each(|desc| desc.set_format(1));
+            row_description
+        } else {
+            self.row_description.clone()
+        };
```
Comment (BowenXiao1999) [inline src/utils/pgwire/src/pg_extended.rs:92]: What is set format 1? If we only have 0 and 1, why not set_to_bin (etc)?

**Review by BowenXiao1999 — COMMENTED** [review]
Rest LGTM

**Review by BowenXiao1999 — APPROVED** [review]

File: src/utils/pgwire/src/pg_extended.rs (hunk 1 above)
Comment (BowenXiao1999) [reply src/utils/pgwire/src/pg_extended.rs]: 0 and 1 is also unnecessary. Maybe just use a bool flag is enough 😄

File: src/common/src/types/chrono_wrapper.rs [hunk 2]
```diff
@@ -86,6 +88,30 @@ impl ToText for NaiveDateTimeWrapper {
...
+    }
+}
+
+impl ToBinary for NaiveDateTimeWrapper {
+    fn to_binary(&self) -> Option<Bytes> {
+        let mut output = BytesMut::new();
+        self.0.to_sql(&Type::ANY, &mut output).unwrap();
+        Some(output.freeze())
+    }
+}
```
Comment (xxchan) [inline src/common/src/types/chrono_wrapper.rs:113]: Why don't we use `implement_using_to_sql` for these types?

**Pull request merged**